angularSTEP/src/app/app.module.ts

```diff
@@ -20,17 +32,23 @@ import { AppRoutingModule } from './app-routing.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { SetupComponent } from './setup/setup.component';
 import { HomeComponent } from './home/home.component';
+>>>>>>> ac803c5992db0cb6f90c3e7dc6dcad7cb70b23d1
```

Comment:
And this shows the version of the file that is in conflict along with which commit introduced the change. These three special marker lines are always grouped together for a given merge conflict. The way it's resolved depends on what the ultimate goal is:
- sometimes the version being merged in is used in its entirety
- sometimes the previous commit in this branch is used in its entirety
- sometimes there are important parts from both that need to be combined manually